packages/webapp/src/elections/api/eden-contract.ts

```diff
@@ -93,15 +95,16 @@ export const getMemberGroupParticipants = async (
     return rows;
 };
 
-const getVoteDataRow = async (
-    opts: VoteDataQueryOptionsByField
+export const getVoteDataRow = async (
+    opts?: VoteDataQueryOptionsByField
```

Comment:
You could make `opts` required here, as this function really _does_ need it.

Then in your `queryVoteDataRow` in `queries.ts`, you can throw there: 

```
export const queryVoteDataRow = (account?: string) => ({
    queryKey: ["query_vote_data_row", account],
    queryFn: () => {
        if (!account) throw new Error("getVoteDataRow requires an account (not undefined)"); // or something like this
        return getVoteDataRow({ fieldName: "member", fieldValue: account });
    },
});
```

Whether you handle `undefined` for `account` in `queryVoteDataRow` or later in `getVoteDataRow`, I recommend you throw and error and not just return `undefined`. React Query expects it. When we use the query in our UI, we'll still get `undefined` for data, but now we'll have a friendly error to go along with it. (And the query is disabled anyways if `account` is `undefined`, so we should never see it.

This would tighten up the types and make it super clear to the developer what's happening instead of just seeing `undefined`.

Reply:
This could apply to `getMemberGroupParticipants` above too.